src/components/NumberInput/NumberInput.js

```diff
@@ -252,43 +255,41 @@ export default class NumberInput extends Component {
     ) : null;
 
     return (
-      <div className={`${prefix}--form-item`}>
```

Comment:
This wrapper `<div>` removal is a breaking change and need to be roped off. As I noted elsewhere, discussion with TJ is nice here - Would we anticipate one `<FormItem>` containing more than one form controls, or usage of this component without form wrapper...?